Add a `session-interaction` event to allow injected js to interact with apps Electron `session` object (#1132)
nativefier/nativefier: docs/api.md

However, there are legitimate use cases to disable such a warning. For example, if you are using Nativefier to ship a kiosk app exposing an internal site (over which you have control). Under those circumstances, it is reasonable to disable this warning that you definitely don't want end-users to see.

More description about the options for `nativefier` can be found at the above [section](#command-line).

## Accessing The Electron Session

> Sometimes there are Electron features that are exposed via the [Electron session api](https://www.electronjs.org/docs/api/session), that may not be exposed via Nativefier options. These can be accessed with an injected javascript file (via the `--inject` command line argument when building your application). Within that javascript file, it may send an ipcRenderer event of `session-interaction` and listen for a `session-interaction-reply` event to get any result. Session properties and functions can be accessed via this event. This event takes an object as an argument with the desired interaction to be performed.

Reviewer: ```suggestion
Sometimes there are Electron features that are exposed via the [Electron `session` API](https://www.electronjs.org/docs/api/session), that may not be exposed via Nativefier options. These can be accessed with an injected javascript file (via the `--inject` command line argument when building your application). Within that javascript file, you may send an ipcRenderer `session-interaction` event, and listen for a `session-interaction-reply` event to get any result. Session properties and functions can be accessed via this event. This event takes an object as an argument with the desired interaction to be performed.
```

> When requesting a session property:

Reviewer: ```suggestion
**Warning**: using this feature in an `--inject` script means using Electron's `session` API, which is not a standard web API and subject to [Breaking Changes](https://www.electronjs.org/docs/breaking-changes).

To get a `session` property:
```
